styles.css

```diff
@@ -0,0 +1,308 @@
+
+.grid-box {
+  display: grid;
+  grid-template-columns: 100%;
+  align-items: center;
+  justify-items: center;
+  width: 100%;
+  margin: 1px;
+}
+
+/* CARDS */
+
+.info-box {
+  background-color: #363237;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  position: relative;
+  margin: 10px 0;
+  width: 90%;
+  height: 400px;
+  box-shadow: 1px 1px 3px #363237,
+  -1px -1px 3px #363237;
+}
+
+.img-box {
+  width: 100%;
+  height: 40%;
+  overflow: hidden;
+}
+
+.card-text {
+  height: 60%;
+  width: 90%;
+  text-align: left;
+}
+
+.card-title {
+  margin-top: 16px;
+  margin-bottom: 0;
+}
+
+.date {
+  margin-top: 5px;
+  font-size: small;
+  opacity: 75%;
+}
+
+.link-box {
+  align-self: center;
+  background-color: #f2979726;
+  width: 96%;
+  height: 50px;
+  margin: 0 2% 0 2%;
+  font-size: small;
+  padding: 1px;
+}
+
+.playlist {
+  font-size: small;
+  margin-bottom: 0;
+}
+
+.playlist-name {
+  margin-top: 5px;
+}
+
+.attachments {
+  margin-bottom: 0;
+}
+
+.num-of-attachments {
+  font-size: x-large;
+  color: #64cdb4;
+  margin: 0;
+}
+
+.img {
+  filter: brightness(60%);
+}
+
+.views {
+  position:absolute;
+  top: 5px;
+  right: 20px;
+}
+
+.view-text {
+  margin:0;
+}
+.view-count {
+  margin: 0;
+  font-size: xx-large;
+}
+
+.more-info {
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  color:  #f29797;
+  background-color: #2D4262;
+  font-size: large;
+  height:40px;
+  width: 40px;
+  border-radius: 20px;
+  border: 1px solid #f29797;
+  text-shadow: 2px 2px 2px #73605B;
+  cursor: pointer;
+}
+
+
+/* TABLET */
+
+@media screen and (min-width: 450px) {
+
+.nav-block {
+  flex-direction: column;
+  align-items: center
+}
+
+.navigation {
+  width: 60%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 20%;
+  padding-right: 20%;
+}
+
+.grid-box {
+  grid-template-columns: 1fr 1fr;
+  width: 100%;
+  margin: 1px;
+}
+
+.info-box {
+  width: 95%;
+}
+
+.site-name {
+  width: 100%;
+  display:flex;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+}
+
+
+/* DESKTOP */
+
+@media screen and (min-width: 1079px) {
+  header {
+    padding: 10px 25px 0 25px;
+  }
+
+  .nav-block {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: center;
+    justify-content: center;
+    padding: 20px 0px;
+  }
+
+  .site-name {
+    width: 50%;
+    display:flex;
+    justify-content: flex-end;
+    margin-bottom: 0;
+  }
+
+  .headline {
+  }
+
+  .navigation {
+    width: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-right: 25%;
+    padding-left: 0;
+  }
+
+  .grid-title {
+    justify-content: flex-start;
+    margin: 20px;
+    margin-top: 30px;
+    font-size: xx-large;
+  }
+
+  .grid-box {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    width: 100%;
+    margin: 1px;
+  }
+
+  .info-box {
+    width: 95%;
+  }
+
+}
+
+
+/* COLORS
+Deep Purple: #363237;
```

Comment:
Your color choices are really pleasing for me! But when I ran WAVE, it seems as if there were some contrast errors which would cause people with different sight abilities to struggle when viewing your website.

Reply:
I see that now and thought I fixed it, will continue trying to play around and find a good way of adjusting this!